src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,3 @@
+databaseChangeLog:
```

Comment:
What is this file about? Is it autogenerated?

Reply:
no, it's liquibase file